components/ws-manager-bridge/src/redis/publisher.ts

```diff
@@ -7,19 +7,33 @@
 import { inject, injectable } from "inversify";
 import { log } from "@gitpod/gitpod-protocol/lib/util/logging";
 import { Metrics } from "../metrics";
+import { RedisClient } from "./client";
+import { WorkspaceInstanceUpdate, WorkspaceInstanceUpdatesChannel } from "@gitpod/gitpod-protocol";
```

Comment:
I was somehow confused by the name `WorkspaceInstanceUpdate` and somehow expecing the full shape (with a similar name?) from ws-manager.
Can we somehow highlight (without making it to bulky) that this is for redis, or just the ids, or similar? :shrug:

Reply:
I could prefix it with Redis: `RedisWorkspaceInstanceUpdate`

Reply:
renamed